### PR TITLE
 TST: remove Python 3.5 from Appveyor [Do Not Merge] [skip travis]

### PR DESCRIPTION
**Pull request by cgohlke**
Fix https://github.com/matplotlib/matplotlib/issues/9176

### CODE REVIEW

File: lib/matplotlib/__init__.py [hunk 1]
```diff
@@ -1347,12 +1347,15 @@ def tk_window_focus():
...
-        faulthandler.enable()
+    # work around https://bugs.python.org/issue31701
+    # faulthandler dumps 'Windows fatal exception: code 0xe06d7363'
+    if not (sys.platform == 'win32' and sys.version_info[:3] == (3, 6, 3)):
+        try:
+            import faulthandler
+        except ImportError:
+            pass
+        else:
+            faulthandler.enable()
```
Comment (Kojoley) [inline lib/matplotlib/__init__.py]: I would remove this completely and enable it on travis with `pytest-faulthandler`.

Comment (anntzer) [reply lib/matplotlib/__init__.py]: even better (optional) if you document it at http://matplotlib.org/devdocs/devel/testing.html#requirements

Comment (cgohlke) [reply lib/matplotlib/__init__.py]: Not sure I follow. Faulthandler is useful, just not the current version or for cases like #9176

Comment (Kojoley) [reply lib/matplotlib/__init__.py]: 1. It's output is not useful if you do not have debug symbols (distutils does not produce it for windows and strips for linux by default on non-debug version)
2. `faulthandler` could be enabled with pytest by `pytest-faulthandler` module (that's what I've suggested earlier)

**Comment by Kojoley** [comment]
We do not upload conda builds from appveyor, right?

File: lib/matplotlib/__init__.py (hunk 1 above)
Comment (cgohlke) [reply lib/matplotlib/__init__.py]: I do have debug symbols and use them frequently when testing matplotlib. `pytest-faulthandler` is yet another dependency. I'll leave it to the maintainers. This PR is to check if the build and tests succeed on Appveyor with Python 3.6.3.

Comment (Kojoley) [reply lib/matplotlib/__init__.py]: The build would pass even without disabling `faulthandler`.

Comment (cgohlke) [reply lib/matplotlib/__init__.py]: Yes, disabling `faulthandler` is not related to building. Still, Python 3.6.3 now uses the vc141 toolchain if installed and I would like to see if that affects matplotlib.

**Comment by anntzer** [comment]
why was this closed?

**Pull request closed without merging**